homework-submission/week1-project/src/Components/todoComponents.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+import React, { Component } from 'react';
+
+class Description extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            onEdit: false,
+        };
+      }
+    
+    handleEdit = (e) => {
+        if(e.target.name === 'edit')
+            this.setState({onEdit: true});
+        else
+        this.setState({onEdit: false});
```

Comment:
what about 
```suggestion
        this.setState({onEdit: !this.state.onEdit});
```
and removing the previous lines?